app/websocket/websocket_app.py

```diff
@@ -1,8 +1,27 @@
 #!/bin/env python
+"""
+    Grid Node is a Socket/HTTP server used to manage / compute data remotely.
+"""
+
 from app import create_app, socketio
 import sys
+import requests
+import json
+import os
+
+# These environment variables must be set before starting the application.
+gateway_url = os.environ["GRID_NETWORK_URL"]
```

Comment:
Can we use argparse instead? https://docs.python.org/3/howto/argparse.html

The default values could be environment variables.

Reply:
Done! (4c90afd )